### PR TITLE
Template for GitHub issues [Finishes#154206112]

### PR DESCRIPTION
**Pull request by evdoks**

### CODE REVIEW

File: .github/issue_template.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+## Logs, error output, etc.
+ 
+ (`epoch.yaml` configuration file, `log/epoch.log` log file plus other related logs, e.g., `log/epoch_mining.log`; if it’s long, please paste to https://ghostbin.com/ and insert the link here)
+ 
+ ...
+
+
+## Specifications
+
+  - Version:
```
Comment (dincho) [inline .github/issue_template.md]: It's not pretty clear to me what Version/Platform/Subsystem is.

**Comment by dincho** [comment]
Could you also make issue_template.md upper case and move it to root for consistency.

Eventually I propose to move all the documents (not only GH related) to `docs` (as GH supports that too). However this would also need change for https://github.com/aeternity/epoch/blob/master/rebar.config#L39

File: .github/issue_template.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+## Specifications
+
+
+  - Virtualization: [*none / metal / container / VM*]
+  - Hardware specs: 
+    * RAM: 
+    * CPU:
+  - OS: [*distribution, version*]
+  - Node Version:
```
Comment (sennui) [inline .github/issue_template.md]: we need to know how the OTP was installed in the system (brew/source/packages) and which OTP is it

Also we need to know how `epoch` was installed (source/packages)

Comment (dincho) [reply .github/issue_template.md]: @sennui I think we should "support" only the official release deployments, our master is not stable enough to support deploys installed from it, is it?

If we agree to support only package installs from official release, then we ship with OTP already. The only external dependancy AFAIK is libssl.

**Pull request merged**